stable/kong/templates/prometheus-plugin.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if and .Values.ingressController.enabled .Values.ingressController.metrics.enabled }}
```

Comment:
As I said in a review before, let's remove this.
The user can create an additional KongPlugin resource if the need be after the Kong Ingress Controller is installed.

Reply:
Do note that Kong will expose some basic observability stats even if this plugin is not enabled.

Reply:
OK, let it be as you're suggesting, I've got the point :)